js/scripts/release.sh

```diff
@@ -34,11 +34,16 @@ git fetch origin 2>$GITLOG
 git checkout -b $BRANCH
 
 echo "*** Committing compiled files for $UTCDATE"
+mv build ../build.new
+git pull origin/$BRANCH -X ours --commit -m "$UTCDATE [merge]"
+git rm build/*.*
```

Comment:
Suggest using `*` instead of `*.*`. Some files might not have a dot in the name.